### PR TITLE
(docs): fixed 'Continuous Integration' typo

### PR DESCRIPTION
**Pull request by 0xflotus**
I only fixed a small typo error

### CODE REVIEW

**Comment by connor-baer** [comment]
Hey @0xflotus, thanks for the fix. Would you mind correcting the typo in the [link hash](https://github.com/sumup-oss/foundry/pull/106/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20) (LOC 20) and [headline](https://github.com/sumup-oss/foundry#-continous-integration-ci) (LOC 142) as well, please?

**Comment by 0xflotus** [comment]
> Hey @0xflotus, thanks for the fix. Would you mind correcting the typo in the [link hash](https://github.com/sumup-oss/foundry/pull/106/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20) (LOC 20) and [headline](https://github.com/sumup-oss/foundry#-continous-integration-ci) (LOC 142) as well, please?

Sure. Done :)

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ A toolkit that makes it a breeze to set up and maintain JavaScript + TypeScript
 - [Presets](#presets)
   - [🔍 Lint](#-lint)
   - [🚀 Release](#-release)
-  - [🤖 Continous Integration (CI)](#-continous-integration-ci)
+  - [🤖 Continuous Integration (CI)](#-continous-integration-ci)
```
Comment (connor-baer) [inline README.md:20]:
```suggestion
  - [🤖 Continuous Integration (CI)](#-continuous-integration-ci)
```

Comment (0xflotus) [reply README.md]: My bad. Now it should be done.

**Pull request merged**